### PR TITLE
Bump version to 0.22.0

### PR DESCRIPTION
**Pull request by kimhanbeom**
- Updated dependency for security vulnerabilities.
- Update the compatibility version of the quic communication modules.

Related issue: #849

### CODE REVIEW

**Comment by sophie-cluml** [comment]
I'd like to suggest including https://github.com/aicers/giganto/pull/848 in the new release, which is currently waiting for an additional review from @sehkone. Would it be possible?

**Review by sophie-cluml — APPROVED** [review]

**Comment by sehkone** [comment]
> I'd like to suggest including #848 in the new release, which is currently waiting for an additional review from @sehkone. Would it be possible?

I think I missed this request. I am sorry about that. But It's ok to proceed anyway.

**Pull request merged**
